build.gradle

```diff
@@ -38,6 +38,10 @@ allprojects {
         sourceCompatibility targetJavaVersion
         targetCompatibility targetJavaVersion
       }
+
+      tasks.withType(JavaCompile).configureEach {
+        options.compilerArgs += ["-Xlint:-options"] // Remove when targeting higher Java version
```

Comment:
What is this for?

Reply:
Java 8 target is marked as deprecated in Java 20. So when the project is compiled by Java 20 it prints an additional warning which I don't believe is actionable here (yet). This particular line disabled the warning. 
Given I reverted to compiling with Java 17 - the warning is gone and I removed this line.